plugin/src/test/groovy/org/gradle/testretry/ParenthesesFuncTest.groovy

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.testretry
+
+import org.gradle.testkit.runner.TaskOutcome
+import org.gradle.util.GradleVersion
+
+class ParenthesesFuncTest extends AbstractPluginFuncTest {
+
+    @Override
+    String getLanguagePlugin() {
+        "org.jetbrains.kotlin.jvm' version '1.9.23"
+    }
+
+    def "should work with parentheses in test name"(String gradleVersion, Tuple2<Closure<File>, String> scenarios) {
+        given:
+        def (setupTest, String testSource) = scenarios
+        setupTest(buildFile)
+
+        and:
+        writeKotlinTestSource testSource
+
+        expect:
+        def result = gradleRunner(gradleVersion, "test").build()
+        result.task(":test").outcome == TaskOutcome.SUCCESS
+
+        where:
+        [gradleVersion, scenarios] << [
+            // Kotlin plugin compatible from 6.8 onwards
+            GRADLE_VERSIONS_UNDER_TEST.findAll { GradleVersion.version(it) >= GradleVersion.version("6.8") },
+            [
+                new Tuple2<>({ bf -> setupJunit5Test(bf) }, junit5TestWithParentheses()),
```

Comment:
```suggestion
                Tuple.tuple({ bf -> setupJunit5Test(bf) }, junit5TestWithParentheses()),
```
maybe with static import `groovy.lang.Tuple.tuple`

Reply:
Done in 4b280bd